src/util/enum.py

```diff
@@ -19,3 +19,11 @@ class Side(Enum):
     CLIENT = 'client'
     SERVER = 'server'
     BOTH = 'both'
+
+
+class Asset(Enum):
```

Comment:
on the comment above, maybe Asset should be an object that encapsulates assets themselves (like each lockfile entry is an asset). This should be something like `AssetType` and is a property of the LockfileEntry.